sdk/trace/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -111,4 +112,18 @@ default InstrumentationScopeInfo getInstrumentationScopeInfo() {
    */
   @Nullable
   <T> T getAttribute(AttributeKey<T> key);
+
+  /**
+   * Returns the Span attributes.
+   *
+   * <p>Attributes can be changed during the lifetime of the Span by using {@link
+   * Span#setAttribute}} so this value cannot be cached.
+   *
+   * <p>Note: the implementation of this method performs locking to ensure thread-safe behavior.
```

Comment:
(sorry, I'm pretty sure my javadoc link format is incorrect)

```suggestion
   * <p>Note: the implementation of this method performs locking and returns an immutable copy to ensure thread-safe behavior. If you only need a single attribute it is better to call {@link #getAttribute(AttributeKey)}.
```